CONTRIBUTING.md

```diff
@@ -59,7 +59,7 @@ Triage member or another Mappings Team member.
 
 **Final Comment Period**:
 - Snapshots: 1 day
-- Releases: 4 days
+- Releases: 3 days
```

Comment:
I'd argue this should be 2 instead of 3, to be consistent with doubling the time over snapshots.